lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -669,6 +669,44 @@ sub _job_stop_children {
     }
 }
 
+sub _carry_over_labels {
+    my ($job) = @_;
+    if ($job->comments > 0) {
+        log_debug("not carrying over labels from previous jobs as there are already comments on current");
+        return;
+    }
+
+    # search for previous jobs
+    my @scenario_keys  = qw/DISTRI VERSION FLAVOR ARCH TEST/;
+    my %js_settings    = map { $_ => $job->settings_hash->{$_} } @scenario_keys;
+    my $subquery       = schema->resultset("JobSettings")->query_for_settings(\%js_settings);
+    my @conds          = [{'me.state' => 'done'}, {id => {'<', $job->id}}, {'me.id' => {-in => $subquery->get_column('job_id')->as_query}}];
+    my $limit_previous = 200;                                                                                                                  # arbitrary limit not to search all jobs
```

Comment:
I'm confused by 200 limit. Do you want to carry comments from older jobs when i.e. the previous one didn't have any comment?